api/example.com/resource/gpu/nas/v1alpha1/client/client.go

```diff
@@ -99,3 +99,22 @@ func (c *Client) Get() error {
 	*c.nas = *crd
 	return nil
 }
+
+func (c *Client) Switch(newcrd *nascrd.NodeAllocationState) {
+	*c.nas = *newcrd
+}
+
+func (c *Client) GetAll() (map[string]*nascrd.NodeAllocationState, error) {
+	naslist, err := c.client.NodeAllocationStates(c.nas.Namespace).List(
+		context.TODO(), metav1.ListOptions{})
```

Comment:
Does this line differ somehow? Why do we wrap?

Reply:
Missed this when unwrapping. Wiill fix with and rebase to renaming change